resources/measures/advanced_rtu_control/measure.rb

```diff
@@ -0,0 +1,335 @@
+        is_unitary_system = true
+      end
+    end
+    return is_unitary_system
+  end
+  
+   #slightly modified from OS standards to adjust log messages 
+   def thermal_zone_outdoor_airflow_rate(thermal_zone)
+    tot_oa_flow_rate = 0.0
+
+    spaces = thermal_zone.spaces.sort
+
+    sum_floor_area = 0.0
+    sum_number_of_people = 0.0
+    sum_volume = 0.0
+
+    # Variables for merging outdoor air
+    any_max_oa_method = false
+    sum_oa_for_people = 0.0
+    sum_oa_for_floor_area = 0.0
+    sum_oa_rate = 0.0
+    sum_oa_for_volume = 0.0
+
+    # Find common variables for the new space
+    spaces.each do |space|
+      floor_area = space.floorArea
+      sum_floor_area += floor_area
+
+      number_of_people = space.numberOfPeople
+      sum_number_of_people += number_of_people
+
+      volume = space.volume
+      sum_volume += volume
+
+      dsn_oa = space.designSpecificationOutdoorAir
+      next if dsn_oa.empty?
+
+      dsn_oa = dsn_oa.get
+
+      # compute outdoor air rates in case we need them
+      oa_for_people = number_of_people * dsn_oa.outdoorAirFlowperPerson
+      oa_for_floor_area = floor_area * dsn_oa.outdoorAirFlowperFloorArea
+      oa_rate = dsn_oa.outdoorAirFlowRate
+      oa_for_volume = volume * dsn_oa.outdoorAirFlowAirChangesperHour / 3600
+
+      # First check if this space uses the Maximum method and other spaces do not
+      if dsn_oa.outdoorAirMethod == 'Maximum'
+        sum_oa_rate += [oa_for_people, oa_for_floor_area, oa_rate, oa_for_volume].max
+      elsif dsn_oa.outdoorAirMethod == 'Sum'
+        sum_oa_for_people += oa_for_people
+        sum_oa_for_floor_area += oa_for_floor_area
+        sum_oa_rate += oa_rate
+        sum_oa_for_volume += oa_for_volume
+      end
+    end
+
+    tot_oa_flow_rate += sum_oa_for_people
+    tot_oa_flow_rate += sum_oa_for_floor_area
+    tot_oa_flow_rate += sum_oa_rate
+    tot_oa_flow_rate += sum_oa_for_volume
+
+    # Convert to cfm
+    tot_oa_flow_rate_cfm = OpenStudio.convert(tot_oa_flow_rate, 'm^3/s', 'cfm').get
+
+    #runner.registerInfo("For #{thermal_zone.name}, design min OA = #{tot_oa_flow_rate_cfm.round} cfm.")
+
+    return tot_oa_flow_rate
+  end
+
+  # define what happens when the measure is run
+  def run(model, runner, user_arguments)
+    super(model, runner, user_arguments)  # Do **NOT** remove this line
+
+    # use the built-in error checking
+    if !runner.validateUserArguments(arguments(model), user_arguments)
+      return false
+    end
+	#read in arguments
+	add_econo = runner.getBoolArgumentValue('add_econo', user_arguments)
+	add_dcv = runner.getBoolArgumentValue('add_dcv', user_arguments)
+	
+	#Set constants
+	fan_tot_eff = 0.7 #Eplus default, reasonable
+    fan_mot_eff = 0.9 #Eplus default, reasonable 
+	
+	#set airflow design ratios
+	max_flow = 0.9
```

Comment:
Does this mean we do not allow 100% design airflow? Is this supported by documentation?

Reply:
Per our discussion, I've reset that to 100%.